homeassistant/components/zeversolar_local/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow for Local access to the zeversolar invertor integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+from zeversolarlocal import api
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN, ZEVER_HOST, ZEVER_INVERTER_ID, ZEVER_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(ZEVER_HOST): str,
+    }
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str, Any]:
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
+    """
+
+    url = api.default_url(data[ZEVER_HOST])
+
+    # If parsing of data is wrong or inverter is
+    # unreachable, this will raise ZeverError.
+    inverter_id = await api.inverter_id(url)
+
+    return {
+        "title": "Zeversolar invertor.",
```

Comment:
Is it possible to get a name from the device to use as config entry title?
The title is used to identify the config entry if you have multiple config entries for the same integration.

Reply:
I'll add the inverter_id to the title

Reply:
https://github.com/home-assistant/core/pull/55440/commits/55a1a307036b152a900150ea953618d28d72e65d